internal/pkg/template/diff/override.go

```diff
@@ -101,28 +231,15 @@ func stripTag(node *yaml.Node) *yaml.Node {
 	}
 }
 
-func intrinsicFuncFullFormName(fullFormNode *yaml.Node) string {
-	return fullFormNode.Content[0].Value
-}
-
-// Explicitly maintain a map so that we don't accidentally match nodes that are not actually intrinsic function
-// but happen to match the "Fn::" and "!" format.
-var intrinsicFuncFull2Short = map[string]string{
-	"Ref":             "!Ref",
-	"Fn::Base64":      "!Base64",
-	"Fn::Cidr":        "!Cidr",
-	"Fn::FindInMap":   "!FindInMap",
-	"Fn::GetAtt":      "!GetAtt",
-	"Fn::GetAZs":      "!GetAZs",
-	"Fn::ImportValue": "!ImportValue",
-	"Fn::Join":        "!Join",
-	"Fn::Select":      "!Select",
-	"Fn::Split":       "!Split",
-	"Fn::Sub":         "!Sub",
-	"Fn::Transform":   "Transform",
-}
-
-func eqIntrinsicFunc(fullFormName, shortFormName string) bool {
-	expectedShort, ok := intrinsicFuncFull2Short[fullFormName]
-	return ok && shortFormName == expectedShort
+// Transform scalar node "LogicalID.Attr" to sequence node [LogicalID, Attr].
+func getAttScalarToSeq(scalarNode *yaml.Node) (*yaml.Node, error) {
```

Comment:
i wonder if it's easier to compare the other direction? (seq -> scalar)? bc then it's just 
```
strings.Join(seqNode.Value, ".")
```
right? or am i missing something 🤔

Reply:
> For the Fn::GetAtt attribute name, you can use the Ref function. [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html#getatt-supported-functions)

It would have been OK if `GetAtt` only accepts strings for the "Attributes" part : (


```yaml
# from
Fn::GetAtt: a.b
# to
Fn::GetAtt:
  - a
  - !Ref BAttr
```

I'm not sure how to convert `[a, !Ref BAttr]` to a scalar 🥲

Reply:
ah, rip. of course, the nested functions :(